eval.py

```diff
@@ -88,21 +91,41 @@ def eval(**args):
     # Setup Model To Evaluate 
     model.eval()
 
+    ret_data   = None
+    ret_labels = None
+
     with torch.no_grad():
         for step, data in enumerate(eval_loader):
             x_input     = data['data'].to(device)
             annotations = data['annots']
 
-            outputs = model(x_input)
+            outputs = model(x_input, features=True)
```

Comment:
This change isn't backwards compatible with other models. I'll change it to be separate keyword argument